### PR TITLE
5.2.1 blog post draft

### PR DESCRIPTION
**Pull request by tst-nfarah**
Open GEE 5.2.1 blog post

### CODE REVIEW

File: docs/_posts/2018-03-20-521-release.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Hi Everyone!
+
+We are happy to bring you the news of Open GEE’s 5.2.1 Release! This is a comprehensive update of Open GEE 5.2.0. It contains new features, several bug fixes in Fusion and GEE Server, 
+and updates to third-party libraries. Some of the main new features are:
+
+**MrSID support**: At build time, Open GEE checks for MrSID libraries and, if found, Fusion will be built with MrSID raster format support.
+
+**RPM Installers**: For RedHat and CentOS, RPMs can now be built to enable easier installation. For Ubuntu, the installer scripts continue to be the supported way to install Open GEE.
+
+**JPEG2000 performance boost**: Updated version of OpenJpeg2000 library has considerable performance increase.
```
Comment (tst-swanson) [inline docs/_posts/2018-03-20-521-release.md]: In description, should Jpeg in OpenJpeg2000 library be capped?

File: docs/_posts/2018-03-20-521-release.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+**Controlling quality of terrain.**: Command-line tool gepackgen producing packets now accepts a decimation value. In general, reducing the decimation value for terrain produces higher quality and better looking terrain.
```
Comment (tst-swanson) [inline docs/_posts/2018-03-20-521-release.md]: higher-quality and better-looking

File: docs/_posts/2018-03-20-521-release.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+**Testing scripts enhanced**: <code>publish_tutorial.sh</code> and <code>publish_historical.sh</code> scripts now publish the pushed databases.
+
+**Detailed version info**: Version number displayed in user interface shows detailed information indicating an official release or a development build.
+
+**Continuous Integration**: Travis CI is used for continuous integration. On every new commit to Open GEE a build is executed to validate the committed changes.
+
+To download this release see the [release page](https://github.com/google/earthenterprise/releases/tag/5.2.1-6.final). The full release notes can also be found [here](http://www.opengee.org/geedocs/answer/7160001.html).
+
+A big thank you to all of the contributors who helped make this release possible! Next release, Open GEE 5.2.2, development is already in progress.
```
Comment (tst-swanson) [inline docs/_posts/2018-03-20-521-release.md]: The next release,

File: docs/_posts/2018-03-20-521-release.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+Would you be interested to be part of the project ? Please join us at <a href="http://slack.opengee.org"> Slack</a> and visit the project <a href="https://github.com/google/earthenterprise">Github repository</a>. We would love to see you there.
```
Comment (tst-swanson) [inline docs/_posts/2018-03-20-521-release.md]: Would you like to be part of the project?
visit the project's

File: docs/_posts/2018-03-20-521-release.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+Hi Everyone!
+
+We are happy to bring you the news of Open GEE’s 5.2.1 Release! This is a comprehensive update of Open GEE 5.2.0. It contains new features, several bug fixes in Fusion and GEE Server, 
+and updates to third-party libraries. Some of the main new features are:
+
+**MrSID support**: At build time, Open GEE checks for MrSID libraries and, if found, Fusion will be built with MrSID raster format support.
+
+**RPM Installers**: For RedHat and CentOS, RPMs can now be built to enable easier installation. For Ubuntu, the installer scripts continue to be the supported way to install Open GEE.
```
[the remark this answers is not in the document]
Comment (tst-dsugar) [reply docs/_posts/2018-03-20-521-release.md]: Well, they do continue to be supported and used for now.

File: docs/_posts/2018-03-20-521-release.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+**JPEG2000 performance boost**: Updated version of OpenJpeg2000 library has considerable performance increase.
+
+**Controlling quality of terrain.**: Command-line tool gepackgen producing packets now accepts a decimation value. In general, reducing the decimation value for terrain produces higher quality and better looking terrain.
+
+**Testing scripts enhanced**: <code>publish_tutorial.sh</code> and <code>publish_historical.sh</code> scripts now publish the pushed databases.
+
+**Detailed version info**: Version number displayed in user interface shows detailed information indicating an official release or a development build.
+
+**Continuous Integration**: Travis CI is used for continuous integration. On every new commit to Open GEE a build is executed to validate the committed changes.
```
[the remark this answers is not in the document]
Comment (tst-dsugar) [reply docs/_posts/2018-03-20-521-release.md]: Good, just wanted to make sure...and that we are consistent.

**Pull request merged**